Replaced the twilio adapter with Generic SMS adaptor (#3)
madebymany/passwordless_auth: README.md

## Configuration

> PasswordlessAuth uses [ExTwilio](https://github.com/danielberkompas/ex_twilio) as a default sms adapter. So if you want to use that then you need to set ExTwilio config in your `config/config.exs` file:

Reviewer: ```suggestion
PasswordlessAuth uses [ExTwilio](https://github.com/danielberkompas/ex_twilio) as a default sms adapter. If you wish to use ExTwilio you'll also need to set the following configuration in your `config/config.exs` file:
```

```elixir
config :passwordless_auth, sms_adapter: ExTwilio

config :ex_twilio,
  account_sid: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN",
  workspace_sid: "TWILIO_WORKSPACE_SID" # optional
```

> Note: We expact that the custome SMS adapter has a module called Message with create function.

Reviewer: ```suggestion
Note: We expect the custom SMS adapter to have a module called Message with a create function.
```
